nixos/modules/services/web-apps/drupal.nix

```
@@ -0,0 +1,157 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    literalExpression
+    mapAttrs
+    mapAttrs'
+    mkDefault
+    mkEnableOption
+    mkIf
+    mkMerge
+    mkOption
+    mkPackageOption
+    nameValuePair
+    types
+    ;
+  cfg = config.services.drupal;
+
+  siteOpts =
+    {
+      options,
+      config,
+      lib,
+      name,
+      ...
+    }:
+    {
+      options = {
+        enable = mkEnableOption "Drupal web application";
+        package = mkPackageOption pkgs "drupal" { };
+
+        stateDir = mkOption {
+          type = types.path;
+          default = "/var/lib/drupal/${name}/data";
```

I believe this is the reason behind the ofborg failure.

```suggestion
          default = "/var/lib/drupal/${name}/data";
          defaultText = "/var/lib/drupal/<name>/data";
```